### PR TITLE
Implemented accept() and close()

### PR DESCRIPTION
**Pull request by cyian-1756**
I implemented accept and close below is a echo server to test with

```lua 
local k=require("socket");
  local s=assert(k.bind("*",7867));
  local c=s:accept();
  while true do
    local r,x=c:receive()
    c:send(r)
  end
  c:close()
  f:close();
```

### CODE REVIEW

File: socketcore/masteraccept.go [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+	"bufio"
+
+	"github.com/yuin/gopher-lua"
+)
+
+func masterAcceptMethod(L *lua.LState) int {
+	master, ud := checkMaster(L)
+	conn, err := master.Listener.Accept()
+	if err != nil {
+		panic(err)
```
Comment (drauschenbach) [inline socketcore/masteraccept.go]: panic? This seems strange.

Comment (cyian-1756) [reply socketcore/masteraccept.go]: Woops, forgot to remove that after debugging

**Comment by mikegray** [comment]
In your example echo server in the PR comment, what is "f:close()"?

**Comment by cyian-1756** [comment]
@mikegray 

> In your example echo server in the PR comment, what is "f:close()"?

It's a typo. I adapted ir from another script and forgot to remove that line

**Pull request merged**